js/modules/k6/experimental/tracing/options_test.go

```diff
@@ -5,11 +5,19 @@ import "testing"
 func TestOptionsValidate(t *testing.T) {
 	t.Parallel()
 
-	testFloat := 10.0
+	// Note that we prefer variables over constants here
+	// as we need to be able to address them.
+	var (
+		validSampling            = 10
+		lowerBoundSampling       = 0
+		upperBoundSampling       = 100
+		lowerOutOfBoundsSampling = -1
+		upperOutOfBoundsSampling = 101
+	)
 
 	type fields struct {
 		Propagator string
-		Sampling   *float64
+		Sampling   *int
```

Comment:
Is this change in type what we want? 

DO we not want to support `12.5%` sampling? Or should that be left for later?

Reply:
This is based on an assumption of mine. I switched to `int` because I had the assumption `float` would lead to a usage of the form `sampling: 0.4` to express 40%, which as a user always confuses me. But I didn't consider the possibility of providing decimals to a percent 🤔

Any ideas, position, or feedback on this @dgzlopes and @Blinkuu? Are you fine with a `sampling` option in the form of a `int` bounded as : `0 <= n <= 100`, or would you prefer a `float64` with the same bounds but supporting decimals? 🙇🏻

Reply:
> would you prefer a float64 with the same bounds but supporting decimals? 🙇🏻

I think I would go with this one :+1:

Reply:
In Grafana Agent, sampling is configured using the following:

```json
 {
    "default_strategy": {
      "param": 1.0,
      "type": "probabilistic"
    }
}
```

In the above, `0.0` is the lower bound and `1.0` is the upper bound.

If we are implementing _probabilistic_ sampling, then I think we should go with the `float64` and `[0.0, 1.0]` range. I've never seen a probability higher than `1.0`.